git_wrapper/clone.py

```diff
@@ -12,17 +11,16 @@
 from git_wrapper import exceptions
 
 
-logger = logging.getLogger(__name__)
-
-
 class GitWrapperClone(GitWrapperBase):
     """Provides git clone functionality"""
 
-    def __init__(self, path='', repo=None):
+    def __init__(self, path='', repo=None, logger=None):
         # If we're cloning for the first time, we may legitimately not
         # have a path or repo to use yet - so don't initialize just yet
         if path or repo:
-            super(GitWrapperClone, self).__init__(path=path, repo=repo)
+            super(GitWrapperClone, self).__init__(path=path, repo=repo, logger=None)
```

Comment:
why not just pass in logger here, and let the super init it as needed?

Reply:
Whoops! Typo, sorry about this.